adaptive/runner.py

```diff
@@ -740,9 +749,16 @@ def shutdown(self, wait=True):
         pass
 
 
+def _default_executor():
+    if with_loky:
+        return loky.get_reusable_executor()
```

Comment:
great idea 👍

Reply:
In principle this is a breaking change, but I reckon it's ok